ginga/AstroImage.py

```diff
@@ -62,7 +61,21 @@ def __init__(self, data_np=None, metadata=None, logger=None,
 
         self.io = ioclass(self.logger)
 
+        # these attributes to be removed--DO NOT USE!!!
+        inherit_primary_header = False
+        if 'inherit_primary_header' in kwargs:
+            warnings.warn("inherit_primary_header kwarg has been deprecated--"
+                          "use save_primary_header when loading image",
+                          PendingDeprecationWarning)
+            inherit_primary_header = kwargs['inherit_primary_header']
         self.inherit_primary_header = inherit_primary_header
+
+        save_primary_header = True
+        if 'save_primary_header' in kwargs:
+            warnings.warn("save_primary_header kwarg has been deprecated--"
+                          "use save_primary_header when loading image",
```

Comment:
I don't understand this warning...

Reply:
Probably needs a better message.  The idea is that when loading a file you can specify `save_primary_header=True` and it will additionally save the primary header in the AstroImage. Because that is what is actually happening--the primary header is not being actually inherited--it's just saved separately.  When you ask for the header (via `get_header()` with `include_primary_header=True` you get a merged header.

Reply:
See what you think of the updated warning messages.